windows/mdag_enable_disable/enable_mdag.yml

```diff
@@ -107,7 +111,8 @@
 
 - name: "Set the result of getting MDAG process"
   ansible.builtin.set_fact:
-    mdag_process_result: "{{ win_powershell_cmd_output }}"
+    mdag_process_running: "{{ true if win_powershell_cmd_output.rc == 0 else false }}"
```

Comment:
Use `{{ win_powershell_cmd_output.rc == 0 }}` is enough.

Reply:
Fixed. Thanks.